number_theory/numeric.h

```diff
@@ -0,0 +1,48 @@
+#ifndef NUMBER_THEORY_NUMERIC_H
+#define NUMBER_THEORY_NUMERIC_H
+
+#include <limits>
+#include <numeric>
+#include <utility>
+
+// Common numeric functions.
+
+namespace number_theory {
+
+using std::gcd;
```

Comment:
I think these should be removed since they are not used yet. They can be introduced in the future commits when they are needed.

Furthermore, I think they should be placed outside of `number_theory` namespace so that we do not pollute the namespace unless we prefer making those functions available in our namespace. However, I don't see a reason to do that since users can simply use them from `std` namespace.

Reply:
I marked the line 12 but it seems GitHub seletected from 2-12 by itself. I was talking about the `using` lines.